practical_09/band.py

```diff
@@ -0,0 +1,29 @@
+"""
+Band class
+"""
+
+
+class Band:
+    """Band class"""
+
+    def __init__(self, name=""):
+        """Construct a Band with a name and empty musician collection."""
+        self.name = name
+        self.musicians = []
+
+    def __str__(self):
+        """Return a string representation of a band."""
+        return f"{self.name} ({', '.join([musician.__str__() for musician in self.musicians])})"
```

Comment:
Is there a reason to use `musician.__str__()` instead of `str(musician)`? I did some reading and apparently there are some subtle differences. What we've seen in class has been the str() form, so I'd prefer that for now.